### PR TITLE
Prom histograms in Flux draft-in-progress

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/2287. Incorporated content from Chris; @sanderson review and updates.

@wolffcm, we added this content to the "Create histograms with Flux" doc. You referred to the `histogramquantile()` function specifically. Is there a distinction to be made between this function and the `histogram()` function in the context of creating histograms for Flux? 

Does it make sense to include links on docs for both of these functions to this new content? Lmk your thoughts-thanks!

### CODE REVIEW

File: content/influxdb/v2.0/query-data/flux/histograms.md [hunk 1]
```diff
@@ -190,3 +190,119 @@ In the Histogram visualization options, select `_time` as the **X Column**
...
+// The windows are then reassembled to produce a single table for each
+// unique group key.
+doQuantile = (tables=<-, q) => tables
+  |> histogramQuantile(quantile: q)
+  |> duplicate(as: "_time", column: "_stop")
+  |> window(every: inf)
+  |> map(fn: (r) => ({r with _measurement: "quantile", _field: string(v: q)}))
+  |> experimental.group(mode: "extend", columns: ["_measurement", "_field"])
+
+histos =
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/flux/histograms.md:278]:
```suggestion
histograms =
```

File: content/influxdb/v2.0/query-data/flux/histograms.md [hunk 2]
```diff
@@ -190,3 +190,119 @@ In the Histogram visualization options, select `_time` as the **X Column**
...
+  // Fields will have names like "0.001", etc. Change _field to a float column called "le".
+  // This also has the effect of ungrouping by _field, so bucket counts for each period
+  // will be within the same table.
+  |> map(fn: (r) => ({r with le: float(v: r._field)}))
+  |> drop(columns: ["_field"])
+
+// Compute the 50th and 95th percentile for duration of http requests.
+union(tables: [
+  histos |> doQuantile(q:  0.95),
+  histos |> doQuantile(q:  0.5)
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/flux/histograms.md:304]:
```suggestion
  histograms |> doQuantile(q:  0.95),
  histograms |> doQuantile(q:  0.5)
```

File: content/influxdb/v2.0/query-data/flux/histograms.md [hunk 3]
```diff
@@ -190,3 +190,119 @@ In the Histogram visualization options, select `_time` as the **X Column**
...
+])
+```
+
+The tables flowing into the call to `histogramQuantile` should look similar to this. Note, that rows are sorted by `le` to be clear that the counts increase for larger upper bounds):
```
Comment (sanderson) [inline content/influxdb/v2.0/query-data/flux/histograms.md:308]:
```suggestion
The tables piped-forward into `histogramQuantile()` should look similar to those returned by the `histograms` variable in the example above. Note, that rows are sorted by `le` to be clear that the counts increase for larger upper bounds.
```

**Review by sanderson — CHANGES_REQUESTED** [review]

**Review by sanderson — APPROVED** [review]

**Review by wolffcm — APPROVED** [review]
@kelseiv @sanderson This looks great!

The `histogram()` function will create histogram data from a metric, similar to the data produced by scraping a prometheus endpoint. As such users can use the guide in this PR to convert it to quantiles with `histogramQuantile()` as well.

I already see a link to this page in the `histogram()` function page in the `Related` section, so think that's good enough.

Thanks for taking the time to fix up my prose!

**Pull request merged**